app/views/vendors.html

```diff
@@ -1,37 +1,123 @@
 <div class="vendor" ng-controller="dimVendorCtrl as vm">
   <a ui-sref='inventory' class="link"><i class='fa fa-arrow-circle-left'></i>Back to DIM</a>
-  <h1 class="orange">Vendors</h1>
-  <div class="vendor">
-    <div class="vendor-container" ng-repeat="activeVendor in vm.activeVendors">
-    <div class="vendor-block" ng-repeat="(vendorType, vendor) in vm.vendors[activeVendor]">
-      <div class="vendor-header">
-        <div class="title">
-        {{vendor.vendorName}}
-        <img class="vendor-icon" ng-src="{{vendor.vendorIcon}}" />
+  <h1>Vendors</h1>
+  <div class="vendor-headers">
+    <div class="store-cell" ng-if="!store.isVault" ng-repeat="store in vm.stores | sortStores:vm.settings.characterOrder track by store.id">
+      <dim-store-heading class="character" store-data="store"></dim-store-heading>
+    </div>
+    <div class="vendor-reset-container">
+      <h2 class="vendor-reset-header">Reset In:</h2>
+    </div>
+  </div>
+  <div class="vendors">
+    <div class="vendor-char-items">
+      <div ng-if="vm.vendors[0][vm.bannerHash]">
+        <div class="vendor-header">
+          <div class="title">
+          {{vm.vendors[0][vm.bannerHash].vendorName}}
+          <img class="vendor-icon" ng-src="{{vm.vendors[0][vm.bannerHash].vendorIcon}}" />
+          <timer ng-if="!vm.vendors[0][vm.bannerHash].nextRefreshDate.startsWith('9999')" style="float:right" end-time="vm.vendors[0][vm.bannerHash].nextRefreshDate" max-time-unit="'day'" interval="1000">{{days}} day{{daysS}} {{hhours}}:{{mminutes}}:{{sseconds}}</timer>
```

Comment:
It's generally bad for performance to call a function in a bound expression. You should also move that style over to the stylesheet.